backend/routes/restaurant.js

```diff
@@ -158,4 +158,42 @@ router.delete('/', (req, res) => {
   });
 });
 
+/**
+ * @swagger
+ *
+ * /restaurant/{restaurantID}/reservations:
+ *   get:
+ *     description: gets a collection of reservations for a given restaurant
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: restaurantID
+ *         description: Primary Key of Restaurant database table
+ *         in: path
+ *         required: true
+ *         type: integer
+ *     responses:
+ *       200:
+ *         description: Successfully retrieved
+ */
+router.get('/:restaurantID/reservations', (req, res) => {
+  const id = req.params.restaurantID;
+
+  if (!id) {
+    res.status(400).json({ error: 'path param: {id} malformed' });
```

Comment:
I believe these should be backticks and a "$" in front of the "{id}"

Reply:
Oops thanks for catching that